src/main/java/io/github/patternatlas/api/rest/controller/DesignModelController.java

```diff
@@ -53,6 +54,7 @@
 import lombok.extern.apachecommons.CommonsLog;
 
 @RestController
+@ConditionalOnExpression(value = "false")  // TODO: set to true if the design models should be used again
```

Comment:
TODO?

Reply:
Design models are currently disabled and their implementation has to be checked/maybe extended or refactored before being enabled again.